src/transformers/adapters/head_utils.py

```diff
@@ -401,6 +401,16 @@
         },
         "layers": {"classifier"},
     },
+    # Electra
```

Comment:
Since Electra also provides other task-specific model classes (e.g. `ElectraForTokenClassification`) in its modeling file, it would be great to also have conversations for those here.